### ISSUE TITLE
remove node_modules folder from repo

### ISSUE THREAD

**Issue by hanayik** [multimodalio/Ments-web#1]
@sammiebenton it's usually best to not upload the node_modules folder (from what I've seen in other repos). I prevent git from uploading the whole folder by adding this to the .gitignore file:

/node_modules

(the above line tells git to ignore the whole folder). When we, or other people run npm install it will read the package.json folder and install everything they, or we need.

**Comment by sammiebenton** [comment]
yeah I already have that .gitignore file configured.  It still uploaded.  fixing now

**Comment by hanayik** [comment]
No worries. I just don't include it so it doesn't interfere with ppl downloading packages from npm
